integrations/argilla.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: integration
+name: Argilla
+description: Use Argilla as a collaboration platform for AI engineers and domain experts!
+authors:
+    - name: Argilla Team
+      socials:
+        github: argilla-io
+        twitter: argilla_io
+        linkedin: argilla-io
+pypi: https://pypi.org/project/argilla-haystack
+repo: https://github.com/argilla-io/argilla-haystack
+type: Monitoring
+report_issue: https://github.com/argilla-io/argilla-haystack/issues
+logo: /logos/argilla.png
+version: Haystack 2.0
```

Comment:
so the biggest issue I see is that this extension doesn't support Haystack 2.0. It was developed using 1.23.

Any chance you'd be up for refactoring it to support 2.0? We'd be happy to help or provide pointers. 2.0 doesn't have agents yet (although it's high-pri and on the roadmap.) 2.0 does support some chat endpoints for models that provide them (Gemini, Mistral, probably others) and it seems like those could be used instead of the Agent responses below. 

Let me know what you're thinking. If you don't have bandwidth to refactor, I can proceed and review it as a 1.x extension. We will be making a big push for 2.0 in the coming months.